fusor-ember-cli/app/styles/openstack.scss

```diff
@@ -288,6 +286,10 @@
   padding: 0;
   display: block;
 }
+.deployment-roles-unassigned .draggable-object {
+  width: 150px;
+  display: inline-block;
+}
```

Comment:
@eriknelson, got it. In the "Files changed" tab, I just saw all the changes in the PR and didn't notice the separate commits. ACK.